cfgov/unprocessed/js/routes/browse-basic/index.js

```diff
@@ -0,0 +1,26 @@
+/* ==========================================================================
+   Scripts for /browse-basic/.
+   ========================================================================== */
+
+'use strict';
+
+// Required polyfills for <IE8.
+require( '../../modules/polyfill/query-selector' );
+
+// List of organisms used.
+var Expandable = require( '../../molecules/Expandable' );
+var ExpandableGroup = require( '../../organisms/ExpandableGroup' );
+
+var expandables = document.querySelectorAll( '.expandable-prototypes .m-expandable' );
```

Comment:
Ah, thanks for the explanation, I should have realized that and was just going to fast. :ok_hand: